common/app/views/mainAMP.scala.html

```diff
@@ -40,6 +42,22 @@
 
             @body
 
+            <div class="fc-container__inner">
+                <div class="fc-container__header__title">
+                    <span>popular</span>
+                </div>
+                <amp-list width="600" height="1000" layout="responsive" src="@if(Play.isDev) {https://www.thegulocal.com} else {@Configuration.ajax.url}/most-read-amp.json">
```

Comment:
does it not work if you just use configuration.ajax.url every time?  I have mine explicitly configured, but I guess not everyone does